app/src/main/res/layout/drawer_header.xml

```diff
@@ -1,50 +1,71 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    style="@style/AppTheme.DrawerHeaderBackground"
     android:layout_width="match_parent"
-    android:layout_height="172dp"
-    android:theme="@style/ThemeOverlay.AppCompat.Dark">
+    android:layout_height="wrap_content"
+    android:background="@color/color_primary"
+    android:fitsSystemWindows="true"
+    android:orientation="vertical"
+    android:theme="@style/ThemeOverlay.AppCompat">
 
-    <View
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:alpha="0.35"
-        android:background="@android:color/black" />
+    <de.hdodenhof.circleimageview.CircleImageView
+        android:id="@+id/profileImageView"
+        android:layout_width="@dimen/icon_large"
+        android:layout_height="@dimen/icon_large"
+        android:layout_marginStart="@dimen/material_default_padding"
+        android:layout_marginTop="@dimen/material_default_padding"
+        android:layout_marginEnd="@dimen/material_default_padding"
+        android:background="@drawable/circle_background"
+        android:src="@drawable/photo_not_available" />
 
-    <LinearLayout
-        android:layout_width="match_parent"
+    <android.support.design.button.MaterialButton
+        android:id="@+id/loginButton"
+        style="@style/Widget.MaterialComponents.Button.OutlinedButton"
```

Comment:
In my quick test this button still showed up, when I was logged in...

Reply:
Oops 🙈